tests/htmlMethods.spec.js

```diff
@@ -15,8 +15,8 @@ describe('Entity without declaration', () => {
     });
 
     it('Should proper attrs', () => {
-        expect(shallow(<Bem block="Block" tag="b" attrs={{ id : 'the-id' }}/>).prop('id'))
-            .toBe('the-id');
+        expect(shallow(<Bem block="Block" tag="b" attrs={{ 'aria-labelledby' : 'address' }}/>).prop('aria-labelledby'))
+            .toBe('address');
```

Comment:
Oh wait, let's ADD tests instead of replacing this with other one

Reply:
We check this case here https://github.com/bem/bem-react-core/blob/master/tests/decl.spec.js
https://github.com/bem/bem-react-core/blob/master/tests/fixtures1.blocks/BlockWithMixins/BlockWithMixins.js

Reply:
Oh! Not this case) Yes, you right) I'm going to return